Libraries/CommonFunctions.psm1

```diff
@@ -2413,4 +2413,28 @@ Function ExtractSSHPublicKeyFromPPKFile {
         } else {
             return $null
         }
+}
+
+function Is-XDPCompatible() {
+    param (
+        [string] $KernelVersion,
+        [string] $DetectedDistro
+    )
+    # ToDo: Update Minimum kernel version check once patches are in downstream distro.
+    $SUPPORTED_DISTRO_KERNEL = @{
```

Comment:
Can you query this from guest, instead of having hard code those distro  kernels?

Reply:
I didn't understand "query from guest" part. Can we keep this information store somewhere?

Reply:
@simonxiaoss This is not from the GOS VM but planned kernel version with XDP. We expect this XDP is available later than this version. So he put this version info here. This should be hard-coded here or put in XML file.

Reply:
I have removed kernel version dependent check and replaced it with xdp feature check which is to check for xdp_drop variable is present in interface statistic or not.